src/events/pollVotationResults.js

```diff
@@ -6,14 +6,27 @@ module.exports = {
   async execute(message) {
     const { embeds, author, client, channelId } = message;
 
-    if (author.bot && embeds[0]?.data?.type !== 'poll_result') {
+    if (
+      author.bot &&
+      !Array.isArray(embeds[0]?.fields) &&
+      embeds[0]?.data?.type !== 'poll_result'
+    ) {
       return;
     }
 
     const userMentionField = embeds[0]?.fields.find(
       (field) => field.name === 'poll_question_text'
     );
+
+    if (userMentionField === undefined) {
+      return;
+    }
+
     const parts = userMentionField?.value.split('|').map((part) => part.trim());
```

Comment:
now we are sure it is not undefined, but we are not sure if value is not undefined, so maybe this optional chain would be better in value
```suggestion
    const parts = userMentionField.value?.split('|').map((part) => part.trim());
```